src/common/legacy_config_opts.h

```diff
@@ -1521,6 +1521,8 @@ OPTION(rgw_list_bucket_min_readahead, OPT_INT) // minimum number of entries to r
 
 OPTION(rgw_rest_getusage_op_compat, OPT_BOOL) // dump description of total stats for s3 GetUsage API
 
+OPTION(rgw_register_with_unique_name, OPT_BOOL) // register in service map with unique name
```

Comment:
i'd suggest stop adding options to this file unless it is accessed in critical path.